<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CumulativeHistogram</class>
 <widget class="QWidget" name="CumulativeHistogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>382</width>
     <height>92</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="meanLabel">
      <property name="text">
       <string>Mean:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="modeLabel">
      <property name="text">
       <string>Mode:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>Count:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="maxLabel">
      <property name="text">
       <string>Max:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="minLabel">
      <property name="text">
       <string>Min:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="stdDevLabel">
      <property name="text">
       <string>StdDev:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QPushButton" name="changeButton">
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>184</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
